probe-rs/targets/MIMXRT685S.yaml

```diff
@@ -153,11 +153,6 @@ variants:
       end: 0x50144000
     cores:
     - cm33
-  flash_algorithms:
```

Comment:
'this no good...

Reply:
Apparently these devices don't declare their flash (I guess because it's external?) so we couldn't flash them previously, and won't be able to flash them now. Only now it's more obvious.